ddl/ddl_api.go

```diff
@@ -3981,6 +3985,15 @@ func setColumnComment(ctx sessionctx.Context, col *table.Column, option *ast.Col
 		return errors.Trace(err)
 	}
 	col.Comment, err = value.ToString()
+	if len(col.Comment) > MaxCommentLength {
```

Comment:
You check table comment and index comment through a separate function, but check the field here? It makes the code hard to maintain. Can you unify them all?